test/PointerCasts/store_cast_char_to_float.ll

```diff
@@ -8,16 +8,20 @@ target triple = "spir-unknown-unknown"
 
 ; CHECK: [[cast:%[a-zA-Z0-9_.]+]] = bitcast float %conv to i32
 ; CHECK: [[shr:%[a-zA-Z0-9_.]+]] = lshr i32 [[cast]], 0
-; CHECK: [[trunc0:%[a-zA-Z0-9_.]+]] = trunc i32 [[shr]] to i8
+; CHECK: [[and0:%[a-zA-Z0-9_.]+]] = and i32 [[shr]], 255
```

Comment:
I'm a bit unsure of the latest change. This mask is unnecessary because it is immediately truncated to 8-bits. Is this mask ever necessary?

Reply:
I don't think the mask is ever necessary; I will remove it.